domains/certificates/Query.API/API.IntegrationTests/ContractTests.cs

```diff
@@ -65,6 +65,30 @@ public async Task CreateContract_ActivateWithEndDate_Created()
         createdContract.Should().BeEquivalentTo(new { GSRN = gsrn, StartDate = startDate, EndDate = endDate });
     }
 
+    [Fact]
+    public async Task CreateContract_ActivateWithEndDateAndConsumptionType_Created()
+    {
+        var gsrn = GsrnHelper.GenerateRandom();
+        dataSyncWireMock.SetupMeteringPointsResponse(gsrn);
```

Comment:
This defaults to be a production metering point.
Think we should remove the default parameter for the metering point type in DataSyncWireMock, so all calls must explicitly state if it is production or consumption

Reply:
Corrected